requirements.txt

```diff
@@ -1,4 +1,5 @@
 bcrypt==3.1.3
+cryptography==1.7.1
```

Comment:
Probably worth a chat with @necrophonic on this, but I'd use the latest version of cryptography available to avoid potential security issues. Latest version is https://pypi.python.org/pypi/cryptography/1.9